testdata/instrumenter/bco/RangeStmt.go

```diff
@@ -0,0 +1,34 @@
+package instrumenter
+
+// https://go.dev/ref/spec#For_statements
+
+// TODO: Add systematic tests.
+
+// rangeStmt covers the instrumentation of [ast.RangeStmt], which has the
+// branch fields Key, Value and X.
```

Comment:
I don't know what you mean by the term *branch fields*. For condition coverage mode, this comment highlights the interesting fields that need to be instrumented further.